plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueConverter.java

```diff
@@ -105,7 +105,8 @@ final class GlueConverter
             StorageFormat.create(
                     "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe",
                     "org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat",
-                    "org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat"),
+                    "org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat",
+                    ".parquet"),
```

Comment:
should we use the constants from `HiveStorageFormat.PARQUET` or does this creates some circular dependency ?